### DISCUSSION TITLE
no hardware control with LG OLED ?

### DISCUSSION THREAD

**Discussion by casi7el** [MonitorControl/MonitorControl#811] [Q&A] [answered]
Sorry for my bad English
My gear: iMac 2019, usb c to hdmi, big sur, LG OLED A1
No hardware control available?

**Comment by waydabber** [comment, marked as answer]
TVs generally do not support hardware control. However you don't miss out anything since OLED has self-lit pixels and does not have a backlighting mechanism to be controlled. The software based gamma table dimming solution (which changes the brightness level of individual pixels) used by MonitorControl actually gives the best results on OLED.

**Reply by casi7el** [reply]
uncheck “use hardware ddc control”
uncheck “avoid gamma table manipulation”

is okay?

**Reply by waydabber** [reply]
Yes.
